skbio/alignment/_pairwise.py

```diff
@@ -258,7 +157,8 @@ def local_pairwise_align_protein(seq1, seq2, gap_open_penalty=11,
                 % type(seq).__name__)
 
     if substitution_matrix is None:
-        substitution_matrix = blosum50
+        substitution_matrix = SubstitutionMatrix.by_name(
+            'BLOSUM50').to_dict()
```

Comment:
Same here re: the use of variables. But again, doesn't need to be in this PR, and may make sense as a separate refactor PR

Reply:
The same as above.